src/content/docs/en/reference/cli-reference.mdx

```diff
@@ -226,6 +226,36 @@ Use these flags to customize the behavior of the command.
 
 The command will watch for any changes in your project, and will report any errors.
 
+#### `--root <path-to-dir>`
+
+Specify a different root directory to check. Uses the current working directory by default.
+
+#### `--tsconfig <path-to-file>`
+
+Specify a `tsconfig.json` or `jsconfig.json` file to use manually. If not provided, Astro will attempt to find a config, or infer the project's config automatically.
+
+#### `--minimumFailingSeverity <error|warning|hint>`
+
+Default: `error`
+
+Minimum severity needed to exit with an error code.
+
+For example, chooisng `warning` will cause the command to exit with an error if any warnings are detected.
+
+#### `--minimumSeverity <error|warning|hint>`
+
+Default: `hint`
+
+Minimum severity to output.
+
+For example, choosing `warning` will show errors and warnings, but not hints.
+
+#### `--preserveWatchOutput`
+
+Default: `false`
+
+If set to false, the output will not be cleared between checks when in watch mode.
```

Comment:
I'm confused about the wording of this one. If the default is false, then why would someone set this flag to false? Do you mean instead that the default behaviour if you do not include this flag is not to clear output between checks? (Not that you run this flag set to false?)

And, does this actually require being run with a `true` or `false` value? Or is just running with this flag the equivalent of specifying whether or not to clear the output between checks when in watch mode?

Is it more like...?

```suggestion
Specifies not to clear the output between checks when in watch mode.
```

Reply:
I made this suggestion to @lifeisfakenews simply quoting 
https://github.com/withastro/language-tools/blob/64a46b81f4fb7b2f6c183cb3aebe211629b87bde/packages/astro-check/src/options.ts#L27C11-L31C4
my bad 😖, should have tested before copypasting it.
<hr>

Clarifying the flag:
`astro check --preserveWatchOutput` which is `true` condition preserves the output on the terminal during subsequent `check`.
Not specifying this flag is the `false` or `default` condition which as said clears the output during subsequent `check`
<hr>

providing `true` or `false` as Sarah asked also works smoothly alongside supported flags but it tends to become verbose (I am turning a blind eye to it, leaving it to fellow reviewers 🙌)
<hr>

Suggestion by Sarah is 👍 🔥